Source/Foundation/bsfCore/Managers/BsGpuProgramManager.cpp

```diff
@@ -96,14 +96,14 @@ namespace bs
 
 	void GpuProgramManager::addFactory(const String& language, GpuProgramFactory* factory)
 	{
-		Lock(mMutex);
```

Comment:
I didn't realize that that syntax would set a new variable. How archaic!

I had assumed that was a macro, for some reason.

Reply:
at the first glance it looks like a `Lock` function call, doesn't it ? :smile: